tools/cli/render.go

```diff
@@ -292,7 +292,7 @@ func trimString(str string, maxLength int) string {
 
 	items := strings.Split(str, "/")
```

Comment:
Your change itself is good but is "/" a standard delimiter in workflow types that we expect/document? If not, this kind of looks arbitrary.

Reply:
It is quite common as workflow type name default to fully qualified name if not overriden. So they have form of `path/to/workflow/file.WorkflowName` to it is get shortened to `.../file.WorkflowName`